scipy/stats/tests/test_multivariate.py

```diff
@@ -637,120 +638,125 @@ def test_moments(self):
                                                         N*num_cols,num_rows).T)
         assert_allclose(sample_rowcov, U, atol=0.1)
 
-class TestDirichlet:
+
+class DirichletTest:
```

Comment:
Will have two subclasses - one with `dist = dirichlet`, the other with `dist = multivariate_beta`.